baseplate/experiments/providers/__init__.py

```diff
@@ -19,6 +22,15 @@
 ISO_DATE_FMT = "%Y-%m-%d"
 
 
+type_class_map = {
+    'r2': R2Experiment,
+    'feature_flag': FeatureFlag,
+    'single_variant': SingleVariantExperiment,
+    'multi_variant': MultiVariantExperiment,
+    'feature_rollout': FeatureRollout,
```

Comment:
maybe this type should be `feature_flag_v2`?  Establish a versioning pattern rather than finding new names for the same thing?

Reply:
The implication with a feature flag is "on/off (for a targeted group)" which isn't the case here. Even if targeted, there's no guarantee that a user will be bucketed in one or the other (unless the rollout percentage is 0 or 100)